Makefile

```diff
@@ -64,8 +66,11 @@ shell:
 run-api-server:
 	go run ./cmd/server/main.go run --port 8080
 
+build-server:
```

Comment:
There are a few differences in `build-server` command between Makefile and Dockerfile, what do you think about unify?